### ISSUE TITLE
Warrior Weapon "Bizzles" anniamation's bug

### ISSUE THREAD

**Issue by Creecreeprs** [Wynncraft/Issues#3266]
So I identified this items 5 minutes ago but their are write is the weapon's description that's is a very slow weapon. But when I use it, i can easly spam click but for the servers, I make very slow attack. Its only an animation bug so its not very important.

(sorry if my english is not optimal but im french :/ )

**Comment by Creecreeprs** [comment]
UPDATE :
When I put the wepon in my enderchest the wepon seems to be right. So I think the bug is because of the identification.

**Comment by Verret001** [comment]
Revised:
So I identified these items five minutes ago but it says in the weapon's description that's is a very slow weapon. But when I use it, I can spam and click fast with it. I make very slow attack. Its only an animation bug so its not very important.

Update: When I put the weapon in the ender-chest the weapon seems to all right. Due to that I think the bug is due to the identification

Recap:
After identifying the weapon, Bizzles I figured out that the animation hits fast but the actually attack and the description says its low, When the item was but in a ender-chest the weapon seemed to look fine. 

Ps @Creecreeprs Im french too :)
